text/0000-rust-governance.md

```diff
@@ -0,0 +1,719 @@
+- Feature Name: not applicable
+- Start Date: 2015-02-27
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes to expand, and make more explicit, Rust's governance
+structure. It seeks to supplement today's core team with several
+*subteams* that are more narrowly focused on specific areas of
+interest.
+
+*Thanks to Nick Cameron, Manish Goregaokar, Yehuda Katz, Niko Matsakis and Dave
+ Herman for many suggestions and discussions along the way.*
+
+# Motivation
+
+Rust's governance has evolved over time, perhaps most dramatically
+with the introduction of the RFC system -- which has itself been
+tweaked many times. RFCs have been a major boon for improving design
+quality and fostering deep, productive discussion. It's something we
+all take pride in.
+
+That said, as Rust has matured, a few growing pains have emerged.
+
+We'll start with a brief review of today's governance and process,
+then discuss what needs to be improved.
+
+## Background: today's governance structure
+
+Rust is governed by a
+[core team](https://github.com/rust-lang/rust/wiki/Note-core-team),
```

Comment:
Broken link, should reference [the wiki backup](https://github.com/rust-lang/rust-wiki-backup/blob/master/Note-core-team.md) instead.